<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgPeriods</class>
 <widget class="QWidget" name="WdgPeriods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="tbAdd">
      <property name="geometry">
       <rect>
        <x>88</x>
        <y>2</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/plus.png</normaloff>:/resources/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbDel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>2</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/minus.png</normaloff>:/resources/minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0000ff;&quot;&gt;Period&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="tbEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="foodcl.qrc">
        <normaloff>:/resources/pen.png</normaloff>:/resources/pen.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="tbMeals">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>1</y>
        <width>38</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>Meals</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvPeriod"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="foodcl.qrc"/>
 </resources>
 <connections/>
</ui>
